### PR TITLE
fix(amazonq): update walkthrough

### PR DESCRIPTION
**Pull request by ivikash**
Notes
- Make inline suggestion as the first item for walkthrough
- Do not redirect user to Try inline suggestion example
- Add CTA and directly open an example
- add example for security scan in walkthrough

### Metrics
```
{
  Metadata: {
    command: '_aws.amazonq.walkthrough.inlineSuggestionsExample',
    duration: '76',
    result: 'Succeeded',
    awsAccount: 'not-set',
    awsRegion: 'us-east-1'
  },
  Value: 1,
  Unit: 'None',
  Passive: true
}
```

### Attachments

https://github.com/aws/aws-toolkit-vscode/assets/371007/73007977-bfae-46f9-a854-593caed1f6c9

### Metrics
```
{
  Metadata: {
    command: '_aws.amazonq.walkthrough.securityScanExample',
    duration: '26',
    result: 'Succeeded',
    awsAccount: 'not-set',
    awsRegion: 'us-east-1'
  },
  Value: 1,
  Unit: 'None',
  Passive: true
}
```

### Attachments

https://github.com/aws/aws-toolkit-vscode/assets/371007/11d60503-24e4-4cb2-97df-44b9035589ad

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by ivikash** [comment]
Screenshot 2024-06-11 at 5 07 53 PM

File: packages/core/src/amazonq/onboardingPage/walkthrough.ts [hunk 1]
```diff
@@ -48,3 +48,23 @@ export const openAmazonQWalkthrough = Commands.declare(`_aws.amazonq.walkthrough
...
+    `_aws.amazonq.walkthrough.inlineSuggestionsExample`,
+    () => async () => {
+        const fileName = 'AmazonQ_generate_suggestion.py'
+        const fileContents = `# TODO: place your cursor at the end of line 5 and press Enter to generate a suggestion.
+# Tip: press tab to accept the suggestion
+
+fake_users = [
+    { "name": "User 1", "id": "user1", "city": "San Francisco", "state": "CA" },`
+
+        const uri = vscode.Uri.parse(`untitled:${fileName}`)
```
Comment (ivikash) [inline packages/core/src/amazonq/onboardingPage/walkthrough.ts:62]: Note: Users will be unable to save the file due to the path.

**Comment by ivikash** [comment]
Updated text to `Scan your current project`.
Screenshot 2024-06-12 at 1 11 41 PM

File: packages/core/src/amazonq/onboardingPage/walkthrough.ts (hunk 1 above)
Comment (justinmk3) [reply packages/core/src/amazonq/onboardingPage/walkthrough.ts:63]: That sounds reasonable and expected

**Review by justinmk3 — APPROVED** [review]

**Pull request merged**